marketplace/emoji-tablet/emojiLib.js

```diff
@@ -0,0 +1,37 @@
+///
+/// emojiLib.js
+/// A library of JSON links for emojis
+/// 
+/// Author: Elisa Lupin-Jimenez
+/// Copyright High Fidelity 2017
+///
+/// Licensed under the Apache 2.0 License
+/// See accompanying license file or http://apache.org/
+///
+/// All assets are under CC Attribution Non-Commerical
+/// http://creativecommons.org/licenses/
+///
+
+module.exports = {
+
+	emojiLib: {
```

Comment:
Since the JSON files won't be changing, we can remove the +Date.now() to allow it to cache. These files should also be named relative to the location rather than hard-coding in S3 bucket links